src/components/screen/ChangePw.tsx

```diff
@@ -38,26 +40,44 @@ function ChangePw(props: Props): ReactElement {
   const [currentPw, setCurrentPw] = useState('');
   const [newPw, setNewPw] = useState('');
   const [confirmPw, setConfirmPw] = useState('');
-  const close = (): void => {
-    navigation.goBack();
-  };
+  const [mutationChangePassword] = useMutation<{ changeEmailPassword: boolean }, MutationChangePasswordInput>(
+    MUTATION_CHANGE_PASSWORD,
+  );
+
+  const navigateBack = (): void => navigation.goBack();
 
   const changePassword = async (): Promise<void> => {
-    if (newPw === confirmPw) {
-      // TODO change password api call
-      Keyboard.dismiss();
-      Alert.alert('', 'Password changed.', [
+    if (newPw !== confirmPw) {
+      Alert.alert(getString('PASSWORD_MUST_MATCH'));
```

Comment:
How about `Alert.alert(getString('ERROR'), getString('....)`?
The long title for Alert dialog is quite ugly.